docker-compose.yml

```diff
@@ -2,12 +2,25 @@ version: "3"
 services:
   redis:
     image: "redis"
```

Comment:
Should we specify exact version of `redis` here? Similarly as it was done in https://github.com/poanetwork/bridge-nodejs/pull/88/files#diff-ac440e3f7ab9ccaac9aeab32ac91b8b3?

Reply:
Yes, it is definitely a good idea to explicitly specify a version of `redis` image.